modules/core/src/v2/wallet.ts

```diff
@@ -2587,4 +2643,144 @@ export class Wallet {
       };
     }
   }
+
+  // #region TSS region
```

Comment:
lets make the below functions private.

alternatively, you can move them to `TssUtils` and make them public since that class will never be exported publicly. this will help with unit testing